disnake/http.py

```diff
@@ -1202,24 +1202,20 @@ def start_thread_in_forum_channel(
         files: Optional[Sequence[File]] = None,
         reason: Optional[str] = None,
         **fields: Any,
-    ) -> Response[threads.Thread]:
-        valid_keys = (
-            # Thread fields
-            "name",
-            "auto_archive_duration",
-            "rate_limit_per_user",
-            "type",
-            # Message fields
+    ) -> Response[threads.ForumThread]:
+        valid_thread_keys = {"name", "auto_archive_duration", "rate_limit_per_user", "type"}
```

Comment:
this should continue to remain a tuple as all other valid_keys

Reply:
https://github.com/DisnakeDev/disnake/pull/504/commits/e827c5209f77a8930809a3f18d0f2436291a74eb